torch_xla/csrc/lowering_context.h

```diff
@@ -110,6 +115,14 @@ class LoweringContext : public torch::lazy::LoweringContext {
       parameters_map_;
   std::vector<xla::XlaOp> root_tuple_;
   OutputMap<xla::XlaOp> emitted_outputs_;
+
+  // Stack frame index tables - we accumulate and write these to the HloModule
+  xla::StackFrameIndexProto indexes_;
```

Comment:
These fields should be unneccesary if we compute the relevant information in Python